plugins/modules/ome_session.py

```diff
@@ -0,0 +1,416 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Dell OpenManage Ansible Modules
+# Version 9.3.0
+# Copyright (C) 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: ome_session
+short_description: Manage OpenManage Enterprise and OpenManage Enterprise modular sessions
+version_added: "9.3.0"
+description:
+  - This module allows you  to create and delete sessions on OpenManage Enterprise and OpenManage Enterprise Modular.
+options:
+  hostname:
+    description:
+      - IP address or hostname of the OpenManage Enterprise.
+    type: str
+  username:
+    description:
+      - Username of the OpenManage Enterprise.
+      - I(username) is required when I(state) is C(present).
+    type: str
+  password:
+    description:
+      - Password of the OpenManage Enterprise.
+      - I(password) is required when I(state) is C(present).
+    type: str
+  port:
+    description:
+      - Port of the OpenManage Enterprise.
+    type: int
+    default: 443
+  validate_certs:
+    description:
+     - If C(false), the SSL certificates will not be validated.
+     - Configure C(false) only on personally controlled sites where self-signed certificates are used.
+    type: bool
+    default: true
+  ca_path:
+    description:
+     - The Privacy Enhanced Mail (PEM) file that contains a CA certificate to be used for the validation.
+    type: path
+  timeout:
+    description:
+     - The HTTPS socket level timeout in seconds.
+    type: int
+    default: 30
+  state:
+    description:
+     - The state of the session in OpenManage Enterprise.
+     - C(present) creates a session.
+     - C(absent) deletes a session.
+     - Module will always report changes found to be applied when I(state) is C(present).
+    choices: [present, absent]
+    type: str
+    default: present
+  x_auth_token:
```

Comment:
Lets add `auth_token` tp keep it consistent with the idrac_session module.

Reply:
Done